src/plugins/kibana_react/public/overview_page/overview_page_actions/overview_page_actions.tsx

```diff
@@ -43,8 +43,8 @@ export const overviewPageActions = ({
         href={addDataHref}
         iconType="plusInCircle"
       >
-        {i18n.translate('kibana-react.kbnOverviewPageHeader.addDataButtonLabel', {
-          defaultMessage: 'Add data',
+        {i18n.translate('kibana-react.kbnOverviewPageHeader.addIntegrationsButtonLabel', {
+          defaultMessage: 'Add integrations',
```

Comment:
This component is only used by `kibana_overview`, but `overview_page_footer`, in the same parent folder, is also used by other apps, so in doubt, I kept this component in `kibana_react`.

Reply:
I hoped we can move it to `overview`, but looks like I missed that `overview_page_footer` is used in other apps :( https://github.com/elastic/kibana/issues/114990